src/messaging/outbound/comms-request/publish-request.js

```
@@ -0,0 +1,24 @@
+import { snsClient } from '../../../messaging/sns/client.js'
+import { publish } from '../../../messaging/sns/publish.js'
+import { config } from '../../../config/index.js'
+import { createLogger } from '../../../logging/logger.js'
+import { buildCommsMessage } from './comms-message.js'
+
+const logger = createLogger()
+const commsSnsTopic = config.get('messaging.commsRequest.topicArn')
+
+const publishCommsRequest = async (payload,recipient) => {
+  
+  try {
+    const sanitizedCommsMessage = await buildCommsMessage(payload,recipient)
+    console.log(sanitizedCommsMessage)
+    await publish(snsClient, commsSnsTopic, sanitizedCommsMessage)
+    logger.info(`Successfully published comms request with ID: ${sanitizedCommsMessage.id}`)
+  } catch (error) {
+    logger.error('Error publishing received message to SNS:', { cause: error })
```

We still need to throw the error here to allow us to return a 500 state. Also should the caller be responsible for logging? If so we probably don't need a try catch here at all...